libraries/ms-drivers/src/x86/pca9539r_gpio_expander.c

```diff
@@ -81,3 +81,13 @@ StatusCode pca9539r_gpio_get_state(const Pca9539rGpioAddress *address,
   *input_state = s_pin_settings[address->i2c_address][address->pin].state;
   return STATUS_CODE_OK;
 }
+#include "log.h"
```

Comment:
You shouldn't need log.h, but if you do, its include should be at the top with the status.h include, not down here.